profile.css

```diff
@@ -5,31 +5,162 @@ body {
 
 }
 
+.container {
+    width: 1200px;
+}
+
 .col-4 {
-    /*position: relative;*/
+    position: relative;
     float: left;
-    width: 33.33%;
-    height: 400px;
     padding: 50px 30px;
-    /*border: 1px solid #FFF;*/
 }
 
 .col-8 {
     float: left;
     width: 66.66%;
-    height: 400px;
+    /*height: 400px;*/
     padding: 50px 10px;
     /*border: 1px solid #FFF;*/
```

Comment:
Remember to either remove this comment or fixing the code in it when you pull into master branch.

Reply:
will do